cli/package.json

```diff
@@ -14,43 +14,32 @@
   },
   "license": "SEE LICENSE IN LICENSE-*",
   "dependencies": {
-    "chalk": "~2.1.0",
-    "command-line-args": "^4.0.2",
-    "command-line-commands": "^2.0.0",
-    "command-line-usage": "^4.0.0",
+    "chalk": "^4.1.0",
+    "command-line-args": "^5.1.1",
+    "command-line-commands": "^3.0.1",
+    "command-line-usage": "^6.1.0",
     "git-config": "0.0.7",
-    "handlebars": "^4.1.0",
-    "inquirer": "^3.0.6",
+    "handlebars": "^4.7.6",
+    "inquirer": "^7.3.2",
     "make-promises-safe": "^5.1.0",
-    "mkdirp": "^0.5.1",
-    "quickly-copy-file": "^1.0.0",
-    "rimraf": "^2.6.1",
-    "rsvp": "^4.6.1",
-    "semver": "^5.1.0",
-    "toml": "^2.3.0",
+    "rimraf": "^3.0.2",
+    "semver": "^7.3.2",
+    "toml": "^3.0.0",
+    "ts-dict": "^0.1.1",
```

Comment:
`ts-dict` can't be dropped because `ts-typed-json` depends on it

Reply:
OK, I published 0.3.0 of `ts-typed-json`, which doesn't use `ts-dict` anymore, it uses `Record` instead. Feel free to try using that instead!